app/helpers/wpcc/your_results_page_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module YourResultsPageHelper
+    TAX_RELIEF_THRESHOLD_RATE = 11_500
+
+    def earning_below_tax_relief_threshold?(salary)
+      salary.to_i < TAX_RELIEF_THRESHOLD_RATE
+    end
+  end
+end
```

Comment:
I foresee a number of situations on the user journey requiring 'helpers'. You've taken the approach to use a helper module. I've been using presenters. We should agree on a standard approach - wdyt? tagging @willhall88 and @tomas-stefano as well.